### ISSUE TITLE
Accessibility: The pagination does not use the current language in ARIA-label.

### ISSUE THREAD

**Issue by MarkoTP** [microsoft-search/pnp-modern-search#4024]
**Version used**
4.13.1

**Describe the bug**
Accessibility:
A few PnP Modern Search issues were identified during the accessibility audit.
Using the PnP Modern Search Results webpart with a screen reader (e.g., NVDA), the pagination does not use the current language in ARIA-label. For example Finnish translation is missing.
The pagination number HTML tag contains::
< a class="active_b75f9440 " href="#" aria-label="Go to page number 1">1< /a>
The Aria-Label is in English even though the user has a browser configured to Finnish and a user profile in Finnish.
WCAG 2.1, 3.1.2

**To Reproduce**
Detailed steps to reproduce the behavior:
Set the user’s language to Finnish both in the browser and in the SharePoint profile.
Configure the search page as you like with search box and results web parts. 
Execute a search that returns filterable results.
Execute a search that returns enough results to paginate multiple pages.

**Expected behavior**
The screen reader (e.g., NVDA) should inform the page number in the user's language.

**Desktop (please complete the following information):**
Browser: Chrome, Version 129.0.6668.90 (Official Build) (64-bit)
